pkgs/servers/mail/mailpit/default.nix

```diff
@@ -30,7 +33,7 @@ let
 
     npmDeps = fetchNpmDeps {
       inherit src;
-      hash = "sha256-5F68ia2V8mw4iPAjSoz0b8z1lplWtAg98BgDXYOmMKs=";
+      hash = versions.npmDepsHash;
```

Comment:
```suggestion
      inherit (source) npmDepsHash;
```

Reply:
I think this wouldn't work, as we need the attribute key `hash`. With your change the key would be `npmDepsHash`.